docs/support_files/api_reference.md

```diff
@@ -37,7 +37,7 @@ Defines a hook which is run after each scenario.
   * `timeout`: A hook-specific timeout, to override the default timeout.
 * `fn`: A function, defined as follows:
   * The first argument will be an object of the form `{sourceLocation: {line, uri}, result: {duration, status, exception?}, pickle}`
```

Comment:
Can we also update for what this argument is since its outdated? Probably fine to just point to the type definition of what we pass in

Reply:
Updated.